src/components/search/type-section/type-section.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Label} from '../../common/label/label.js';
+import {Button} from '../../common/button/button.js'
+
+const TypeSection = (props) => (
+  <div className={'search-type-section'}>
+    <Label content={'search by'} labelUC={true} labelWhite={true}/>
+    {
+      props.searchTypes.list.map((item) => (
+          <Button btnRed={item === props.searchTypes.active}
+                  btnGrey={item !== props.searchTypes.active}
+                  key = {item}
+                  id = {item}
+                  content = {item}
+                  btnClick={props.searchTypeCallback}
+                  btnInline = {true}
+                  btnUC={true}
+          />
+        )
+      )
+    }
+  </div>
+);
+
+TypeSection.propTypes = {
+  searchTypes: PropTypes.object,
```

Comment:
For objects please use `PropTypes.shape()`. 
Example, 
`  optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  }),`

Reply:
ok